frontend/packages/knative-plugin/src/utils/get-knative-resources.ts

```diff
@@ -56,3 +58,27 @@ export const getKnativeServingServices = (dc: K8sResourceKind, props): KnativeIt
   const ksservices = props && props.ksservices ? getKsResource(dc, props.ksservices) : undefined;
   return ksservices && ksservices.length > 0 ? { ksservices } : undefined;
 };
+
+const getEventSourceResource = (
+  dc: K8sResourceKind,
```

Comment:
Should this be a `dc` since there is no deployment config in eventing or serving?

Reply:
I know the example above this uses it, just not sure it is right for either one.

Reply:
Yes this utils work with respective deployment as a param, so dc should be fine here . WDYT

Reply:
ok, we can keep it